dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java

```diff
@@ -97,6 +97,15 @@ public static void requireFATRunner(String className) {
                                             "class level.");
     }
 
+    @Override
+    protected String testName(FrameworkMethod method) {
+        String testName = method.getName();
```

Comment:
instead of fully overriding the default impl of `testName()`, lets instead do `String testName = super.testName()` and then append the repeat action to that.  In case other parts of code hook into the super impl in some way.